SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -66,6 +61,22 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 
 	OnPropertyChangedDelegateHandle = FCoreUObjectDelegates::OnObjectPropertyChanged.AddRaw(this, &FSpatialGDKEditorToolbarModule::OnPropertyChanged);
 	bStopSpatialOnExit = GetDefault<USpatialGDKEditorSettings>()->bStopSpatialOnExit;
+
+	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
+	LocalDeploymentManager = GDKServices.GetLocalDeploymentManager();
+
+	// Bind the play button delegate to starting a local spatial deployment.
+	if (!UEditorEngine::TryStartSpatialDeployment.IsBound())
+	{
+		UEditorEngine::TryStartSpatialDeployment.BindLambda([this]
+		{
+			if (GetDefault<UGeneralProjectSettings>()->bSpatialNetworking)
```

Comment:
Do we handle the situation where this gets toggled off whilst the deployment is running?

Reply:
Hmmm, I think it's best to not unbind the lambda if spatial networking is disabled. How about adding a `bSpatialNetworking` in `VerifyAndStartDeployment()` so even if it's called it doesn't do anything?

Reply:
Added the check, let me know. Otherwise we need to unbind and rebind the lambda as part of the GUI interaction to bSpatialNetworking or we check every tick.